example_config/example_config_base.yaml

```diff
@@ -177,6 +177,17 @@ ses:
 challenge_url:
   enabled: true
 
+# This section specifies which roles consoleme is allowed to pull, cache & display. There may still be workarounds via
+# cURL to create requests directly against consoleme. If you have to block cURL requests for requests against specific
+# roles, it's probably best to enforce IAM permissions for consoleme to only modify specific roles. You should only
+# need one or the other parameter defined below
+# Parameters:
+#  allowed_tags: bool: if true, consoleme will only consider roles tagged with key=consoleme value=managed
+#  allowed_arns: list[string]:  if non-empty, consoleme will only consider the role arns in this list
+roles:
+  allowed_tags: false
```

Comment:
I think going by syntax patterns, it would make most sense for `allowed_tags` to be a list of keys, rather than `"consoleme: managed"` as the only allowed tag, in case people want to specify specific allowed tags.